bundle/store_test.go

```diff
@@ -500,28 +500,35 @@ func TestDeltaBundleLifecycle(t *testing.T) {
 		Value: map[string]string{"name": "alice"},
 	}
 
-	// replace a value
+	// insert value in array using add
 	p4 := PatchOperation{
+		Op:    "add",
+		Path:  "/a/x/1",
+		Value: map[string]string{"name": "mallory"},
+	}
```

Comment:
Can we add a few more test cases where a new node is created for an array and object use case. It would be good if we could cover the scenarios for the `add` operation as described in [RFC 6902](https://datatracker.ietf.org/doc/html/rfc6902).

Reply:
I added the remaining tests and also improved how test operations are organized in the code so I didn't have to renumber a million variables.